src/core/agent/services/signifyNotificationService.ts

```diff
@@ -367,6 +378,32 @@ class SignifyNotificationService extends AgentService {
         await this.markNotification(notif.i);
         return;
       }
+      const notifications = await this.notificationStorage.findAllByQuery({
+        exnSaid: previousExnGrantMsg.exn.d,
+      });
+
+      if (notifications.length) {
+        const notificationRecord = notifications[0];
+        if (
+          !notificationRecord.linkedGroupRequests ||
```

Comment:
I think we can remove this line based on last 2 comments (related to `{}` being the default in DB) 

But please verify this on SQLite too as well as browser

Reply:
still to be removed